transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/PhoneWidget.java

```diff
@@ -83,8 +84,13 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
                 @Override
                 public void onTextChanged(CharSequence s, int start, int before, int count) {
                     if (before != count) {
-                        mValue = s.toString();
+                        mValue = formatToApi(s.toString());
                         mListener.saveTextChanged(getName(), getValue());
+                        if (mField.getMask() != null) {
```

Comment:
I don't see this condition in the other widgets. Any specific reason?

Reply:
oops, forgot to remove it. Realized the if-statement is not necessary since `formatToDisplay()` already does this null check